cmd/thyme/main.go

```diff
@@ -3,36 +3,61 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"log"
 	"os"
 	"runtime"
 
-	"github.com/jessevdk/go-flags"
+	"github.com/kovetskiy/godocs"
```

Comment:
This repo seems to be a custom fork of https://github.com/docopt/docopt.go.

Are we sure that we want to use this instead of docopt.go?